src/snappy.c

```diff
@@ -67,6 +67,35 @@
 #define inline __inline
 #endif
 
+static inline u64 get_unaligned64(const void *b)
```

Comment:
What's the reason this patch differs from the upstream patch?
For future maintenance's sake it'd be better if they were identical.

Reply:
> What's the reason this patch differs from the upstream patch?

MSVC does not support Statement Expressions [1], so they had been replaced with
new static inline functions.

  [1]: https://gcc.gnu.org/onlinedocs/gcc/Statement-Exprs.html#Statement-Exprs

>For future maintenance's sake it'd be better if they were identical.

Indeed, but it is already differs, and the reason is again MCSV/win32

Reply:
@edenhill friendly ping on this patch